python/tests/test_pyvw.py

```diff
@@ -20,6 +20,7 @@ def test_learn_predict(self):
         assert init == 0
         self.model.learn(ex)
         assert self.model.predict(ex) > init
+        assert self.model.predict({'x': ['b']}) == 0
```

Comment:
This assertion is tangential to the test. Either remove it or add it in its own test

Reply:
Sure!